ledger/ledger.go

```diff
@@ -302,20 +296,32 @@ func openLedgerDB(dbPathPrefix string, dbMem bool) (trackerDBs trackerdb.Tracker
 		}
 	}
 
-	trackerDBFilename = dbPathPrefix + ".tracker.sqlite"
-	blockDBFilename = dbPathPrefix + ".block.sqlite"
-
 	outErr := make(chan error, 2)
 	go func() {
 		var lerr error
-		trackerDBs, lerr = sqlitedriver.OpenTrackerSQLStore(trackerDBFilename, dbMem)
+		switch cfg.StorageEngine {
+		case "sqlite":
+			fallthrough
+		// anything else will initialize a sqlite engine.
+		default:
+			file := dbPathPrefix + ".tracker.sqlite"
+			trackerDBs, lerr = sqlitedriver.Open(file, dbMem, log)
+		}
+
```

Comment:
makes sense we wouldn't have `pebbledb` here *yet*, but the comment is a little confusing, "anything else will initialize a sqlite engine"